### PR TITLE
improve portability

### PR DESCRIPTION
**Pull request by Habbie**

### CODE REVIEW

**Comment by Habbie** [comment]
squashed, ready for review/merge

File: .travis.yml [hunk 1]
```diff
@@ -80,6 +81,7 @@ before_script:
  - sudo chmod 0777 /var/lib/softhsm
  - p11-kit -l # ensure it's ok
 script:
+ - env
```
Comment (pieterlexis) [inline .travis.yml]: Is this on purpose or was this only for debugging?

Comment (Habbie) [inline .travis.yml]: debugging, will fix

**Comment by Habbie** [comment]
rebased, debugging line gone

**Comment by pieterlexis** [comment]
Looks like it indeed  does not functionally changes the tests and is more portable. If travis is happy, I'm happy. :+1:

**Pull request merged**